### PR TITLE
Fix e2e tests that found issues, adding pricing page review

### PR DESCRIPTION
**Pull request by amurKontur**
## Summary by CodeRabbit

- **New Features**
	- Enhanced login tests for the 'atlas' project, including checks for text availability on the pricing page and navigation to the profile page.
	- Introduced a new `PricingPage` class to manage interactions with the pricing page, including visibility checks for key elements and links.
	- Added error handling for known issues in user rights and page reload functionality for the 'atlas' project.

- **Bug Fixes**
	- Improved test reliability by ensuring all necessary elements are checked concurrently on the pricing page.
	- Marked known issues in the test suite to enhance clarity regarding the status of tests.

- **Configuration Updates**
	- Adjusted timeout settings for Playwright tests to allow for longer wait times in both CI and local environments.

### CODE REVIEW

File: e2e/page-objects/pricingPage.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        await this.page.getByText('Book a demo', { exact: true }).getAttribute('href'),
+      ).toEqual('https://calendly.com/kbakhanko/atlas-demo'),
+      expect(
+        await this.page
+          .getByText('Request trial', { exact: true })
+          .first()
+          .getAttribute('href'),
+      ).toEqual('https://www.kontur.io/demo-call/'),
+      expect(this.page.getByText('Contact sales', { exact: true })).toBeVisible(),
+      plansAndPricingTexts.forEach((text) => expect(text).toBeVisible()),
```
[the remark this answers is not in the document]
Comment (amurKontur) [reply e2e/page-objects/pricingPage.ts:18]: I run them inside of Promise.all

File: e2e/page-objects/pricingPage.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      expect(this.page.getByText('Save 5%', { exact: true })).toBeVisible(),
+    ]);
```
[the remark this answers is not in the document]
Comment (amurKontur) [reply e2e/page-objects/pricingPage.ts]: I want to run all checks at the same time. I don't want to run them sequentually. It lets tests run faster

File: e2e/page-objects/pricingPage.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      expect(this.page.getByText('Professional', { exact: true })).toBeVisible(),
+      expect(this.page.getByText('Educational', { exact: true })).toBeVisible(),
+      expect(this.page.getByText('Custom', { exact: true })).toBeVisible(),
+      expect(this.page.getByText('Book a demo', { exact: true })).toHaveAttribute(
+        'href',
+        'https://calendly.com/kbakhanko/atlas-demo',
+      ),
+      expect(
+        this.page.getByText('Request trial', { exact: true }).first(),
+      ).toHaveAttribute('href', 'https://www.kontur.io/demo-call/'),
```
[the remark this answers is not in the document]
Comment (amurKontur) [reply e2e/page-objects/pricingPage.ts]: good point but no sense as this method is once written and used at different places. you break KISS principle

File: e2e/registration.spec.ts [hunk 4]
```diff
@@ -72,6 +72,12 @@ for (const project of projects) {
       shouldSuccess: true,
     });
 
+    // Atlas redirects to pricing page after login
+    if (project.name === 'atlas') {
+      await pageManager.atPricingPage.checkPageAndTextsAvailability();
+      await pageManager.atNavigationMenu.clickButtonToOpenPage('Profile');
+    }
+
```
[the remark this answers is not in the document]
Comment (amurKontur) [reply e2e/registration.spec.ts]: it is a test

**Review by Natallia-Harshunova — APPROVED** [review]

File: e2e/registration.spec.ts [hunk 5]
```diff
@@ -72,6 +72,22 @@ for (const project of projects) {
...
     });
 
+    // Atlas redirects to pricing page after login
+    if (project.name === 'atlas') {
+      await pageManager.atPricingPage.checkPageAndTextsAvailability();
+      await pageManager.atPricingPage.clickBtnAndAssertUrl({
+        context,
+        buttonName: 'Request trial',
+        expectedUrlPart: 'demo-call',
+      });
```
[the remark this answers is not in the document]
Comment (amurKontur) [reply e2e/registration.spec.ts:82]: they are used just once, don't break kiss

**Pull request merged**